pom.xml

```diff
@@ -210,6 +231,95 @@
         </plugin>
       </plugins>
     </pluginManagement>
+
+
   </build>
 
+    <profiles>
+        <profile>
+            <id>jdk11</id>
+            <properties>
+                <javac.src.version>1.11</javac.src.version>
+                <javac.target.version>1.11</javac.target.version>
+
+                <jdk.version>1.11</jdk.version>
+                <maven.compiler.source>1.11</maven.compiler.source>
+                <maven.compiler.target>1.11</maven.compiler.target>
+                <maven.compiler.release>1.11</maven.compiler.release>
+            </properties>
+            <!-- Not Necessary, but useful-->
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <version>2.22.0</version>
+                        <configuration>
+                            <skipTests>false</skipTests>
+                            <argLine>
+                                --illegal-access=permit
+                            </argLine>
+                            <excludes>
+                                <exclude>com/fasterxml/jackson/**/failing/*.java</exclude>
+                            </excludes>
+                        </configuration>
+                    </plugin>
+                    <!-- Yes.. -->
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-javadoc-plugin</artifactId>
+                        <version>3.0.1</version>
+                        <configuration>
+                            <detectJavaApiLink>false</detectJavaApiLink>
+                            <offlineLinks>
+                                <offlineLink>
+                                    <url>https://docs.oracle.com/javase/${maven.compiler.release}/docs/api/</url>
+                                    <location>${project.basedir}</location>
+                                </offlineLink>
+                            </offlineLinks>
+                            <failOnError>false</failOnError>
+                        </configuration>
+                    </plugin>
+                    <!-- Allows you to switch builds quickly and make sure it all works-->
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>build-helper-maven-plugin</artifactId>
+                        <version>1.7</version>
```

Comment:
Surprisingly no property for version yet, but `oss-parent` does specify 1.7 to be used.